### ISSUE TITLE
Cant run on synology nas

### ISSUE THREAD

**Issue by kimpig** [marcopeocchi/yt-dlp-web-ui#4]
Hello, before I go in, this could be my problem.

I am using Docker on Synology NAS.

However, when I run it, an X is displayed at the bottom and there is no response even when I press the start button.
Is there anything I set wrong?

![image](https://user-images.githubusercontent.com/61234733/156871126-05b11845-706e-47c5-9023-4a6a59d9fe82.png)
![image](https://user-images.githubusercontent.com/61234733/156871137-ba11a26d-c41a-42aa-bbd2-7559f4268dd9.png)
![image](https://user-images.githubusercontent.com/61234733/156871254-3b334ccb-ded1-46b5-a7b4-8ece3077ce6a.png)

**Comment by marcobaobao** [comment]
Hello! This could be due to the fact that the local storage is empty, as you can see there's no server address specified (it says not specified). I will release a fix for this.

In the meantime you can set the local IP address of your NAS In the settings (instead of localhost). E.g. my NAS is located by 192 168.1.200 so on the first run I had to changed it in the settings.

**Comment by kimpig** [comment]
I changed localhost to my NAS IP address (192.168.0.201) in settings but it still didn't work. Is this my problem?

**Comment by marcobaobao** [comment]
Ok, if the process spawns (with no errors) and nothing happens on page it need further investigations.

I will suggest you to wait a little more to see any progess to show on page. yt-dlp takes a lot of time to fire-up especially on low-power hardware or on rather slow mechanical storage. 

In my case i have a nas with a quad-core intel atom and from the time i press start to the time i see the progress on page it takes 10-15 seconds; on a Xeon E5 but with an HDD takes from 5 to 8 seconds (even if it has a way powerful cpu); on my laptop with a ryzen 5 and nvme storage it's pratically istantaneous.

TLDR: fast storage plays a key role.

**Comment by deluxghost** [comment]
we need to check the exit code and print output of ytdlp, otherwise its hard to debug ytdlp side problems

**Comment by archtaurus** [comment]
My synology is 918+, and this image works fine on it.
